### DISCUSSION TITLE
Is there a way to add a custom bar layer to a line chart?

### DISCUSSION THREAD

**Discussion by pluiee** [plouc/nivo#1863] [Q&A]
Issue #139's example(https://codesandbox.io/s/m4ro13jjn8) shows how to add a custom line layer to a bar chart. Are there any methods for adding a custom bar layer to a line chart?

**Comment by plouc** [comment]
The line chart also supports extra layers, so you could use a similar technique, although as for the example you shared, it requires a bit of manual work and D3 knowledge to achieve.

**Reply by jamesarosen** [reply]
In addition to the manual layer to render the line, there would need to be a way to insert the line's name into the legend.

**Comment by alissaVrk** [comment]
[here](https://codesandbox.io/s/nivo-composed-bar-with-line-j6qce5?file=/src/index.js) is an example (also line added to bar and not the other way around) with different axes on the left and on the right.

I think that exporting `useLine` and similar methods would have made it easier
